modules/ingester/instance.go

```diff
@@ -159,6 +166,12 @@ func (i *instance) Push(ctx context.Context, req *tempopb.PushRequest) error {
 
 // PushBytes is used to push an unmarshalled tempopb.Trace to the instance
 func (i *instance) PushBytes(ctx context.Context, id []byte, traceBytes []byte, searchData []byte) error {
+	// measure received bytes as sum of slice capacities
+	// type byte is guaranteed to be 1 byte in size
+	// ref: https://golang.org/ref/spec#Size_and_alignment_guarantees
+	size := cap(id) + cap(traceBytes) + cap(searchData)
```

Comment:
use len() (not cap) and don't count the id, just the traceBytes/searchData

Reply:
I wasn't sure which one to use. Changed, thanks